infra/blueprint-test/pkg/golden/golden.go

```diff
@@ -156,3 +157,16 @@ func (g *GoldenFile) JSONEq(a *assert.Assertions, got gjson.Result, jsonPath str
 	gfData := gf.Get(jsonPath).String()
 	a.Equal(gfData, gotData, fmt.Sprintf("expected %s to match fixture %s", jsonPath, gfData))
 }
+
+// JSONEqs asserts that json content in jsonPaths for got and goldenfile are the same
+func (g *GoldenFile) JSONEqs(a *assert.Assertions, got gjson.Result, jsonPaths []string) {
```

Comment:
nit: Maybe JSONPathEqs might be a better name?

Reply:
SG